docs/services/rook-services-guide.md

```diff
@@ -0,0 +1,652 @@
+          - ""
+          resources:
+          - services
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - apps
+          resources:
+          - statefulsets
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - yugabytedb.rook.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+  - metadata:
+      name: rook-serviceaccount
+      labels:
+        operator: rook-yugabyte
+    spec:
+      template:
+        apiVersion: v1
+        kind: ServiceAccount
+        metadata:
+          name: rook-yugabytedb-operator
+          namespace: rook-yugabytedb-system
+  - metadata:
+      name: rook-serviceaccount
+      labels:
+        operator: rook-yugabyte
+    spec:
+      template:
+        apiVersion: rbac.authorization.k8s.io/v1beta1
+        kind: ClusterRoleBinding
+        metadata:
+          name: rook-yugabytedb-operator
+          namespace: rook-yugabytedb-system
+        roleRef:
+          apiGroup: rbac.authorization.k8s.io
+          kind: ClusterRole
+          name: rook-yugabytedb-operator
+        subjects:
+        - kind: ServiceAccount
+          name: rook-yugabytedb-operator
+          namespace: rook-yugabytedb-system
+  - metadata:
+      name: rook-serviceaccount
+      labels:
+        operator: rook-yugabyte
+    spec:
+      template:
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: rook-yugabytedb-operator
+          namespace: rook-yugabytedb-system
+          labels:
+            app: rook-yugabytedb-operator
+        spec:
+          selector:
+            matchLabels:
+              app: rook-yugabytedb-operator
+          replicas: 1
+          template:
+            metadata:
+              labels:
+                app: rook-yugabytedb-operator
+            spec:
+              serviceAccountName: rook-yugabytedb-operator
+              containers:
+              - name: rook-yugabytedb-operator
+                image: rook/yugabytedb:master
+                args: ["yugabytedb", "operator"]
+                env:
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+EOF
+
+kubectl apply -f rook-operator.yaml
+```
+
+While this is quite a large set of configuration, all it is doing is taking the
+Rook Yugabyte [operator YAML] and packaging it into a Crossplane
+`KubernetesApplication` resource so that we can deploy it into our newly created
+GKE cluster.
+
+!! TODO: check successful provisioning (to be updated after CRD defs are finalized) !! 
```

Comment:
These `TODO`'s will be updated after all the PR's mentioned in the PR body are merged so I can show what actual output will look like.